src/_img/v8-release-80/pointer-compression-chart.svg

```diff
@@ -0,0 +1 @@
 393.09375 25.503124Q393.25 25.565624 394.03125 25.76875Q395.15625 26.08125 395.59375 26.26875Q396.03125 26.45625 396.28125 26.815624Q396.53125 27.175 396.53125 27.721874Q396.53125 28.253124 396.21875 28.721874Q395.92188 29.175 395.34375 29.440624Q394.76562 29.690624 394.03125 29.690624Q392.8125 29.690624 392.17188 29.190624Q391.54688 28.675 391.375 27.690624ZM397.375 27.690624L398.40625 27.534374Q398.5 28.159374 398.89062 28.4875Q399.29688 28.815624 400.03125 28.815624Q400.75 28.815624 401.09375 28.534374Q401.45312 28.2375 401.45312 27.83125Q401.45312 27.471874 401.14062 27.26875Q400.92188 27.128124 400.0625 26.909374Q398.90625 26.6125 398.45312 26.409374Q398.01562 26.190624 397.78125 25.815624Q397.54688 25.440624 397.54688 24.971874Q397.54688 24.565624 397.73438 24.20625Q397.92188 23.846874 398.25 23.6125Q398.5 23.440624 398.92188 23.315624Q399.34375 23.190624 399.84375 23.190624Q400.5625 23.190624 401.10938 23.409374Q401.67188 23.6125 401.9375 23.971874Q402.20312 24.33125 402.29688 24.925L401.26562 25.065624Q401.20312 24.596874 400.85938 24.33125Q400.53125 24.05 399.90625 24.05Q399.1875 24.05 398.875 24.3Q398.5625 24.534374 398.5625 24.8625Q398.5625 25.065624 398.6875 25.221874Q398.82812 25.39375 399.09375 25.503124Q399.25 25.565624 400.03125 25.76875Q401.15625 26.08125 401.59375 26.26875Q402.03125 26.45625 402.28125 26.815624Q402.53125 27.175 402.53125 27.721874Q402.53125 28.253124 402.21875 28.721874Q401.92188 29.175 401.34375 29.440624Q400.76562 29.690624 400.03125 29.690624Q398.8125 29.690624 398.17188 29.190624Q397.54688 28.675 397.375 27.690624ZM403.79688 22.175L403.79688 20.95625L404.85938 20.95625L404.85938 22.175L403.79688 22.175ZM403.79688 29.55L403.79688 23.33125L404.85938 23.33125L404.85938 29.55L403.79688 29.55ZM406.40625 26.440624Q406.40625 24.70625 407.35938 23.878124Q408.15625 23.190624 409.3125 23.190624Q410.59375 23.190624 411.40625 24.034374Q412.23438 24.8625 412.23438 26.346874Q412.23438 27.55 411.875 28.2375Q411.51562 28.925 410.8125 29.315624Q410.125 29.690624 409.3125 29.690624Q408.01562 29.690624 407.20312 28.8625Q406.40625 28.01875 406.40625 26.440624ZM407.48438 26.440624Q407.48438 27.628124 408.0 28.221874Q408.53125 28.815624 409.3125 28.815624Q410.10938 28.815624 410.625 28.221874Q411.14062 27.628124 411.14062 26.409374Q411.14062 25.253124 410.60938 24.659374Q410.09375 24.065624 409.3125 24.065624Q408.53125 24.065624 408.0 24.659374Q407.48438 25.2375 407.48438 26.440624ZM413.79688 29.55L413.79688 23.33125L414.73438 23.33125L414.73438 24.20625Q415.42188 23.190624 416.71875 23.190624Q417.28125 23.190624 417.75 23.39375Q418.23438 23.596874 418.46875 23.925Q418.70312 24.253124 418.79688 24.690624Q418.84375 24.9875 418.84375 25.721874L418.84375 29.55L417.79688 29.55L417.79688 25.76875Q417.79688 25.1125 417.67188 24.8Q417.54688 24.4875 417.23438 24.3Q416.92188 24.096874 416.5 24.096874Q415.82812 24.096874 415.32812 24.534374Q414.84375 24.95625 414.84375 26.14375L414.84375 29.55L413.79688 29.55Z" fill-rule="nonzero"/></svg>
```

Comment:
Please run this file through [SVGOMG](https://jakearchibald.github.io/svgomg/). Even with default settings it cuts the size in half!

Reply:
Ew, text as paths. Turned it into real text, size dropped by a bit more than half ;)